### ISSUE TITLE
Filter option not working on List Component

### ISSUE THREAD

**Issue by LeandroMendesDS** [OfficeDev/microsoft-teams-ui-component-library#71]
Reproduction

When using a List component with the indicated colummns key in the filter prop. Clicking the filter results in 3 different errors.

Where im clicking:
![image](https://user-images.githubusercontent.com/78216924/115541390-f8544b00-a296-11eb-8682-65e0d1c1b1ad.png)

My props:
![image](https://user-images.githubusercontent.com/78216924/115542172-ea52fa00-a297-11eb-8082-bb5f68688cab.png)

Errors:
**1.**

```
Uncaught TypeError: Cannot read property 'pendingProps' of undefined
    at hasAutofocusProp (VM8 0.chunk.js:99355)
    at Object.hasAutofocusFilter [as acceptNode] (VM8 0.chunk.js:99359)
    at VM8 0.chunk.js:99381
    at invokePassiveEffectCreate (VM8 0.chunk.js:390184)
    at HTMLUnknownElement.callCallback (VM8 0.chunk.js:370798)
    at Object.invokeGuardedCallbackDev (VM8 0.chunk.js:370847)
    at invokeGuardedCallback (VM8 0.chunk.js:370907)
    at flushPassiveEffectsImpl (VM8 0.chunk.js:390266)
    at unstable_runWithPriority (VM8 0.chunk.js:438037)
    at runWithPriority$1 (VM8 0.chunk.js:378204)
    at flushPassiveEffects (VM8 0.chunk.js:390143)
    at performSyncWorkOnRoot (VM8 0.chunk.js:388983)
    at VM8 0.chunk.js:378258
    at unstable_runWithPriority (VM8 0.chunk.js:438037)
    at runWithPriority$1 (VM8 0.chunk.js:378204)
    at flushSyncCallbackQueueImpl (VM8 0.chunk.js:378253)
    at flushSyncCallbackQueue (VM8 0.chunk.js:378241)
    at discreteUpdates$1 (VM8 0.chunk.js:389135)
    at discreteUpdates (VM8 0.chunk.js:370608)
    at dispatchDiscreteEvent (VM8 0.chunk.js:372790)
```

**2.**

```
0.chunk.js:394422 The above error occurred in the <Popper> component:

    at Popper (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:99478:23)
    at Transition (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:412865:30)
    at Animation (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:54465:100)
    at PortalInner (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:71368:67)
    at Popup (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:70426:100)
    at ToolbarFilter (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:111485:22)
    at div
    at Box (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:40671:27)
    at div
    at Box (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:40671:27)
    at div
    at RendererProvider (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:265025:7)
    at Provider (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:52745:15)
    at Provider (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:71496:26)
    at ToolbarTheme (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:111854:24)
    at ProviderConsumer (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:71662:24)
    at Toolbar (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:111153:28)
    at List (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:110112:66)
    at div
    at BlockUi (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:364178:5)
    at div
    at Teste (https://2245bdb4cc42.ngrok.io/static/js/main.chunk.js:1268:82)
    at GuardedRoute (https://2245bdb4cc42.ngrok.io/static/js/main.chunk.js:344:24)
    at Switch (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:410764:29)
    at div
    at RendererProvider (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:265025:7)
    at Provider (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:52745:15)
    at Provider (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:71496:26)
    at Router (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:410197:30)
    at ConnectedRouter (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:211929:7)
    at ConnectedRouterWithContext (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:212034:25)
    at Connect(ConnectedRouterWithContext) (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:406765:75)
    at App (https://2245bdb4cc42.ngrok.io/static/js/main.chunk.js:199:22)
    at Provider (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:406478:20)
    at div
    at RendererProvider (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:265025:7)
    at Provider (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:52745:15)
    at Provider (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:71496:26)
    at HVCThemeProvider (https://2245bdb4cc42.ngrok.io/static/js/0.chunk.js:112427:21)
```

**3.**
```
Uncaught TypeError: Cannot read property 'pendingProps' of undefined
    at hasAutofocusProp (VM8 0.chunk.js:99355)
    at Object.hasAutofocusFilter [as acceptNode] (VM8 0.chunk.js:99359)
    at VM8 0.chunk.js:99381
    at invokePassiveEffectCreate (VM8 0.chunk.js:390184)
    at HTMLUnknownElement.callCallback (VM8 0.chunk.js:370798)
    at Object.invokeGuardedCallbackDev (VM8 0.chunk.js:370847)
    at invokeGuardedCallback (VM8 0.chunk.js:370907)
    at flushPassiveEffectsImpl (VM8 0.chunk.js:390266)
    at unstable_runWithPriority (VM8 0.chunk.js:438037)
    at runWithPriority$1 (VM8 0.chunk.js:378204)
    at flushPassiveEffects (VM8 0.chunk.js:390143)
    at performSyncWorkOnRoot (VM8 0.chunk.js:388983)
    at VM8 0.chunk.js:378258
    at unstable_runWithPriority (VM8 0.chunk.js:438037)
    at runWithPriority$1 (VM8 0.chunk.js:378204)
    at flushSyncCallbackQueueImpl (VM8 0.chunk.js:378253)
    at flushSyncCallbackQueue (VM8 0.chunk.js:378241)
    at discreteUpdates$1 (VM8 0.chunk.js:389135)
    at discreteUpdates (VM8 0.chunk.js:370608)
    at dispatchDiscreteEvent (VM8 0.chunk.js:372790)

Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://reactjs.org/link/error-boundaries to learn more about error boundaries.
```

**Comment by thure** [comment]
Hi @LeandroMendesDS — could you please provide your props as text in a [code block](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#quoting-code)? That would help us reproduce your issue more quickly.

**Comment by LeandroMendesDS** [comment]
Sure ! But just heads im doing it dynamically.  
I just inserted the methods used to achieve this logic , if you need the full component let me know


`  

 const fecthAllSign = async () => {
    var fetchOptions = {
      headers: {
        Authorization: `Bearer ${userState.token}`,
        "Content-Type": "application/json",
      },
    };

    try {
      let { data } = await axios.get(
        `${REACT_APP_API_DOMAIN}/api/DocumentList/DocumentsToSign/${userId}`,
        fetchOptions
      );

      var resultado = data.map((dados, indexNumber) => {
        var RowObject = {
          c1: dados.document.name,
          c2: dados.status,
          c3: dados.createdUtc,
          c4: dados.dueDate,
          actions: {
            see: { title: "See", icon: "Edit" },
            sign: { title: "Sign", icon: "Edit", multi: true },
          },
        };

        return RowObject;
      });
      setResponse(resultado);
      setRawData(data);

      setBlockUI(false);
    } catch (error: any) {
      console.log("erro catch aqui");

      console.log(error);
      setBlockUI(false);
    }
  };



const listConfig: any = {
    find: false,
    filters: ["c2"],
    emptySelectionActionGroups: {
    },
    selectable: true,
    columns: {
      c1: {
        title: "Nome do documento",
        sortable: "alphabetical",
      },
      c2: {
        title: "Estado",
      },
      c3: {
        title: "Data de criação",
      },
      c4: {
        title: "Data Limite",
      },
    },
    rows: response,
  };

(....)

<List
            {...listConfig}
            onInteraction={(dados) => ListActionHandler(dados)}
/>

`

Since im waiting for the result of the api , this is  a sample of the result that the variable response will be filled
`{
    "find": false,
    "emptySelectionActionGroups": {},
    "selectable": true,
    "columns": {
        "c1": {
            "title": "Nome do documento",
            "sortable": "alphabetical"
        },
        "c2": {
            "title": "Estado"
        },
        "c3": {
            "title": "Data de criação"
        },
        "c4": {
            "title": "Data Limite"
        }
    },
    "rows": [
        {
            "c1": "E0600DJZI9.pdf",
            "c2": "PREPARADO",
            "c3": "31/03/2021",
            "c4": "2021-04-08",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "trrtr.PDF",
            "c2": "PREPARADO",
            "c3": "31/03/2021",
            "c4": "2021-05-08",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "yupu.PDF",
            "c2": "PREPARADO",
            "c3": "31/03/2021",
            "c4": "2021-04-06",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "teste.PDF",
            "c2": "PREPARADO",
            "c3": "31/03/2021",
            "c4": "2021-04-08",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "teste.PDF",
            "c2": "PREPARADO",
            "c3": "31/03/2021",
            "c4": "4/15/2021 12:00:00 AM",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "A Sample PDF.pdf",
            "c2": "PREPARADO",
            "c3": "31/03/2021",
            "c4": "21-04-15",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "teste.PDF",
            "c2": "PREPARADO",
            "c3": "31/03/2021",
            "c4": "2021-03-31",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "sample.pdf",
            "c2": "PREPARADO",
            "c3": "30/03/2021",
            "c4": "2021-03-31",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "sample.pdf",
            "c2": "PREPARADO",
            "c3": "30/03/2021",
            "c4": "2021-03-31",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "sample.pdf",
            "c2": "PREPARADO",
            "c3": "29/03/2021",
            "c4": "2021-03-31",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "sample.pdf",
            "c2": "PREPARADO",
            "c3": "29/03/2021",
            "c4": "2021-03-31",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "A17_FlightPlan.pdf",
            "c2": "PREPARADO",
            "c3": "29/03/2021",
            "c4": "2021-03-31",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "A17_FlightPlan.pdf",
            "c2": "PREPARADO",
            "c3": "29/03/2021",
            "c4": "2021-03-31",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "A17_FlightPlan.pdf",
            "c2": "PREPARADO",
            "c3": "29/03/2021",
            "c4": "2021-03-31",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "A17_FlightPlan.pdf",
            "c2": "PREPARADO",
            "c3": "29/03/2021",
            "c4": "2021-03-31",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "A17_FlightPlan.pdf",
            "c2": "PREPARADO",
            "c3": "29/03/2021",
            "c4": "2021-03-31",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "A17_FlightPlan.pdf",
            "c2": "PREPARADO",
            "c3": "26/03/2021",
            "c4": "2021-03-30",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "A17_FlightPlan.pdf",
            "c2": "PREPARADO",
            "c3": "26/03/2021",
            "c4": "2021-03-27",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "DevScope PDF signed.pdf",
            "c2": "PREPARADO",
            "c3": "25/03/2021",
            "c4": "2021-03-25",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "DevScope PDF.pdf",
            "c2": "PREPARADO",
            "c3": "25/03/2021",
            "c4": "2021-03-30",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "DevScope PDF.pdf",
            "c2": "PREPARADO",
            "c3": "25/03/2021",
            "c4": "2021-03-25",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "DevScope PDF.pdf",
            "c2": "PREPARADO",
            "c3": "25/03/2021",
            "c4": "2021-03-25",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "sample.pdf",
            "c2": "PREPARADO",
            "c3": "24/03/2021",
            "c4": "2021-03-31",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "insights - portal.pdf",
            "c2": "PREPARADO",
            "c3": "24/03/2021",
            "c4": "2021-03-30",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "insights - portal.pdf",
            "c2": "PREPARADO",
            "c3": "24/03/2021",
            "c4": "2021-03-30",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "A17_FlightPlan.pdf",
            "c2": "PREPARADO",
            "c3": "24/03/2021",
            "c4": "2021-03-30",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "testedelete (4).PDF",
            "c2": "PREPARADO",
            "c3": "22/03/2021",
            "c4": "2021-03-24",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "dummyTeste.pdf",
            "c2": "PREPARADO",
            "c3": "20/04/2021",
            "c4": "2021-04-23",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "testedelete (4).PDF",
            "c2": "PREPARADO",
            "c3": "19/03/2021",
            "c4": "2021-03-25",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "testedelete (4).PDF",
            "c2": "PREPARADO",
            "c3": "19/03/2021",
            "c4": "2021-03-25",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "A Sample PDF.pdf",
            "c2": "PREPARADO",
            "c3": "19/03/2021",
            "c4": "2021-03-31",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "A Sample PDF.pdf",
            "c2": "PREPARADO",
            "c3": "19/03/2021",
            "c4": "2021-03-31",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "A Sample PDF.pdf",
            "c2": "PREPARADO",
            "c3": "19/03/2021",
            "c4": "2021-03-31",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "A Sample PDF.pdf",
            "c2": "PREPARADO",
            "c3": "19/03/2021",
            "c4": "2021-03-31",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "A Sample PDF.pdf",
            "c2": "PREPARADO",
            "c3": "19/03/2021",
            "c4": "2021-03-31",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "A Sample PDF.pdf",
            "c2": "PREPARADO",
            "c3": "18/03/2021",
            "c4": "2021-03-25",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "dummyTeste.pdf",
            "c2": "PREPARADO",
            "c3": "16/04/2021",
            "c4": "2021-04-22",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "A Sample PDF.pdf",
            "c2": "PREPARADO",
            "c3": "16/03/2021",
            "c4": "2021-03-31",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "A Sample PDF.pdf",
            "c2": "PREPARADO",
            "c3": "16/03/2021",
            "c4": "2021-03-31",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "A Sample PDF.pdf",
            "c2": "PREPARADO",
            "c3": "16/03/2021",
            "c4": "2021-03-31",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "dummyTeste.pdf",
            "c2": "PREPARADO",
            "c3": "14/04/2021",
            "c4": "2021-04-29",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "dummyTeste.pdf",
            "c2": "PREPARADO",
            "c3": "14/04/2021",
            "c4": "2021-04-14",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "dummyTeste.pdf",
            "c2": "PREPARADO",
            "c3": "13/04/2021",
            "c4": "2021-04-13",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "dummyTeste.pdf",
            "c2": "PREPARADO",
            "c3": "13/04/2021",
            "c4": "2021-04-28",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "dummyTeste.pdf",
            "c2": "PREPARADO",
            "c3": "13/04/2021",
            "c4": "2021-04-21",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "dummyTeste.pdf",
            "c2": "PREPARADO",
            "c3": "13/04/2021",
            "c4": "2021-04-29",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "dummyTeste.pdf",
            "c2": "PREPARADO",
            "c3": "13/04/2021",
            "c4": "2021-04-29",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "dummyTeste.pdf",
            "c2": "PREPARADO",
            "c3": "13/04/2021",
            "c4": "2021-04-14",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "dummyTeste.pdf",
            "c2": "PREPARADO",
            "c3": "12/04/2021",
            "c4": "2021-04-12",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "dummyTeste.pdf",
            "c2": "PREPARADO",
            "c3": "12/04/2021",
            "c4": "2021-04-29",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "dummyTeste.pdf",
            "c2": "PREPARADO",
            "c3": "12/04/2021",
            "c4": "2021-04-22",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "dummyTeste.pdf",
            "c2": "PREPARADO",
            "c3": "12/04/2021",
            "c4": "2021-04-12",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "dummyTeste.pdf",
            "c2": "PREPARADO",
            "c3": "12/04/2021",
            "c4": "2021-04-12",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "dummyTeste.pdf",
            "c2": "PREPARADO",
            "c3": "09/04/2021",
            "c4": "2021-04-30",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "dummyTeste.pdf",
            "c2": "PREPARADO",
            "c3": "09/04/2021",
            "c4": "2021-04-09",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "dummyTeste.pdf",
            "c2": "PREPARADO",
            "c3": "09/04/2021",
            "c4": "2021-04-22",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "dummyTeste.pdf",
            "c2": "PREPARADO",
            "c3": "08/04/2021",
            "c4": "2021-04-08",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "Basic-Non-Disclosure-Agreement.pdf",
            "c2": "PREPARADO",
            "c3": "08/04/2021",
            "c4": "2021-05-08",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "dummyTeste.pdf",
            "c2": "PREPARADO",
            "c3": "08/04/2021",
            "c4": "2021-04-28",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "Basic-Non-Disclosure-Agreement.pdf",
            "c2": "PREPARADO",
            "c3": "07/04/2021",
            "c4": "2021-05-08",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "E0600DJZI9.pdf",
            "c2": "PREPARADO",
            "c3": "07/04/2021",
            "c4": "2021-05-08",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "dummyTeste.pdf",
            "c2": "PREPARADO",
            "c3": "07/04/2021",
            "c4": "2021-04-29",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "E0600DJZI9.pdf",
            "c2": "PREPARADO",
            "c3": "06/04/2021",
            "c4": "2021-04-14",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "sample.pdf",
            "c2": "PREPARADO",
            "c3": "05/04/2021",
            "c4": "2021-04-23",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "A17_FlightPlan.pdf",
            "c2": "PREPARADO",
            "c3": "05/04/2021",
            "c4": "2021-04-29",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "A17_FlightPlan.pdf",
            "c2": "PREPARADO",
            "c3": "05/04/2021",
            "c4": "2021-04-29",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "sample.pdf",
            "c2": "PREPARADO",
            "c3": "05/04/2021",
            "c4": "2021-04-30",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "dummyTeste.pdf",
            "c2": "PREPARADO",
            "c3": "05/04/2021",
            "c4": "2021-04-21",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "sample.pdf",
            "c2": "PREPARADO",
            "c3": "05/04/2021",
            "c4": "2021-04-21",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "sample.pdf",
            "c2": "PREPARADO",
            "c3": "05/04/2021",
            "c4": "2021-04-29",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "A17_FlightPlan.pdf",
            "c2": "PREPARADO",
            "c3": "05/04/2021",
            "c4": "2021-04-29",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "E0600DJZI9.pdf",
            "c2": "PREPARADO",
            "c3": "01/04/2021",
            "c4": "2021-05-08",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        },
        {
            "c1": "E0600DJZI9.pdf",
            "c2": "PREPARADO",
            "c3": "01/04/2021",
            "c4": "2021-05-06",
            "actions": {
                "see": {
                    "title": "See",
                    "icon": "Edit"
                },
                "sign": {
                    "title": "Sign",
                    "icon": "Edit",
                    "multi": true
                }
            }
        }
    ]
}`

**Comment by LeandroMendesDS** [comment]
Seems related to this [https://github.com/microsoft/fluentui/issues/16192](url)

**Comment by thure** [comment]
I see. This might be resolved by our upgrade to Fluent UI v0.54 in #69 if that's true. I'll also double-check these props in `main` and in that PR to confirm.

**Comment by LeandroMendesDS** [comment]
Updating react Northstar and UI Kit fixed the problem for me.
